docs/evaluation/how-to-evals/evaluating-phoenix-traces.md

```diff
@@ -0,0 +1,196 @@
+# Overview
+This guide will walk you through the process of evaluating traces captured in Phoenix, and exporting the results to the Phoenix UI.
+
+This process is similar to the [evaluation quickstart guide](https://docs.arize.com/phoenix/evaluation/evals), but instead of creating your own dataset or using an existing external one, you'll export a trace dataset from Phoenix and log the evaluation results to Phoenix.
```

Comment:
```suggestion
This builds on the [evaluation quickstart guide](https://docs.arize.com/phoenix/evaluation/evals), but instead of creating your own dataset or using an existing external one, you'll query traces from Phoenix to create a dataset and log the evaluation results to Phoenix so that the traces are annotated with the evaluations.
```